core/lib/zksync_core/src/state_keeper/keeper.rs

```diff
@@ -60,15 +61,15 @@ pub struct ZkSyncStateKeeper {
     stop_receiver: watch::Receiver<bool>,
     io: Box<dyn StateKeeperIO>,
     batch_executor_base: Box<dyn L1BatchExecutorBuilder>,
```

Comment:
Bikeshedding / off-topic (?): I feel that it makes sense to shorten this name to `L1BatchExecutor` (and, naturally, shorten names for the related types introduced in this PR correspondingly, so that we don't need to deal with enterprise™ names like `MainNodeBatchExecutorBuilderLayer`).

Reply:
That's a good idea, but I think I'll ship it as a separate PR to not pack too many things together.